src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamCertificateTrustTests.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.IO.Tests;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using System.Linq;
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    using Configuration = System.Net.Test.Common.Configuration;
+
+    public class SslStreamCertificateTrustTest
+    {
+        [Fact]
+        // not supported on Windows, not implemented elsewhere
+        [PlatformSpecific(TestPlatforms.Linux | TestPlatforms.OSX)]
+        public async Task SslStream_SendCertificateTrust_CertificateCollection()
+        {
+            (X509Certificate2 certificate, X509Certificate2Collection caCerts) = TestHelper.GenerateCertificates("foo");
```

Comment:
nit: you can use nameof(SslStream_SendCertificateTrust_CertificateCollection) -> using test names sometimes helps with debugging if tests are running in parallel.